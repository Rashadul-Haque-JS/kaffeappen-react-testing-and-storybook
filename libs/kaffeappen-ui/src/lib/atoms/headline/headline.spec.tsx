import { render } from '@testing-library/react';
import HeadlineComponent from './headline';


describe('Button', () => {
  it('should render successfully', () => {
    const { baseElement } = render(<HeadlineComponent>Kaffeappen</HeadlineComponent>);
    expect(baseElement).toBeTruthy();
  });

  

  it('should match snapshot successfully', () => {
    const { baseElement } = render(<HeadlineComponent>kaffeappen</HeadlineComponent>);
    expect(baseElement).toMatchSnapshot();
  });

  
});