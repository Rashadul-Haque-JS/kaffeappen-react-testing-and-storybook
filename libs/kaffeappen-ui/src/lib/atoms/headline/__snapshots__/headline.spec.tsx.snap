// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button should match snapshot successfully 1`] = `
<body>
  <div>
    <h1
      class="headline__StyledHeadline-mv29hz-0 hyaiEn"
    >
      kaffeappen
    </h1>
  </div>
</body>
`;
